culinary-canvas/capstone-project/backend/index.js

```diff
@@ -22,44 +24,67 @@ app.use(cors(
     }
 ))
 
-//to create a new user's account
+//to fetch a random image for users's profile
```

Comment:
Question - is the random image only for early-stage development? Or is this as part of the 'required' feature?
I thought it would be like:
- V1: user can pick an image from `Pexels`
- V2: user can upload an image (technical challenge)

Reply:
The random image is for early-stage development.